README.md

```diff
@@ -56,26 +56,26 @@ Yes! *She's Coding* is built by the community, for the community. We currently h
 
 	+ Install [Ruby](https://www.ruby-lang.org/en/):
       ```
-      $ rbenv install 2.3.1
+      $ rbenv install 2.5.3
       ```
 
 	+ Check ruby version:
       ```
       $ ruby -v
       ```
-   	You should see ruby version printed as result: `ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]`
 
+   	You should see ruby version printed as result: `ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]`
+    
+      
 
    ##### Set ruby version if you already have rbenv installed:
    + Change your ruby version locally inside shescoding-dot-org repository:
 
       ```
       $ cd shescoding-dot-org
-      $ rbenv local 2.3.1
+      $ rbenv local 2.5.3
```

Comment:
I used to have `rbenv` as pkg manager, this didn't work - they may have changed the naming